openmdao/core/tests/test_fd_color.py

```diff
@@ -518,110 +519,135 @@ def tearDown(self):
         except OSError:
             pass
 
-    @parameterized.expand(itertools.product(
-        ['fd', 'cs'],
-        [1,2,19],
-        [1,2,11],
-        [True, False]
-        ), name_func=_test_func_name
-    )
-    def test_simple_semitotals(self, method, isplit, osplit, sparse_partials):
-        prob = Problem(coloring_dir=self.tempdir)
-        model = prob.model
+    # semi-total coloring feature disabled.
 
-        sparsity = setup_sparsity(_BIGMASK)
-        indeps, conns = setup_indeps(isplit, _BIGMASK.shape[1], 'indeps', 'sub.comp')
+    #@parameterized.expand(itertools.product(
+        #['fd', 'cs'],
+        #[1,2,19],
+        #[1,2,11],
+        #[True, False]
+        #), name_func=_test_func_name
+    #)
+    #def test_simple_semitotals(self, method, isplit, osplit, sparse_partials):
 
-        model.add_subsystem('indeps', indeps)
-        sub = model.add_subsystem('sub', CounterGroup())
-        sub.declare_coloring('*', method=method)
-        comp = sub.add_subsystem('comp', SparseCompExplicit(sparsity, method, isplit=isplit, osplit=osplit,
-                                                            sparse_partials=sparse_partials))
+        #prob = Problem(coloring_dir=self.tempdir)
+        #model = prob.model
 
-        for conn in conns:
-            model.connect(*conn)
+        #sparsity = setup_sparsity(_BIGMASK)
+        #indeps, conns = setup_indeps(isplit, _BIGMASK.shape[1], 'indeps', 'sub.comp')
 
-        for i in range(isplit):
-            model.add_design_var('indeps.x%d' % i)
+        #model.add_subsystem('indeps', indeps)
```

Comment:
Lots of commented code blocks.  I'm assuming this is done so that it can be easily reimplemented when fixed, but we don't have a timetable for that.  Should we just use @skip decorators or remove it entirely?

Reply:
@robfalck : Yeah, that was a little messy. I've added skiptests instead.